plugin.php

```diff
@@ -135,6 +135,32 @@
 	return $pDI->get(EstateViewDocumentTitleBuilder::class)->buildDocumentTitle($title);
 }, 10, 2);
 
+add_filter('get_post_metadata', function($value, $object_id, $meta_key) use ($pDI) {
+	$pDataDetailViewHandler = $pDI->get(DataDetailViewHandler::class);
+	$pDetailView = $pDataDetailViewHandler->getDetailView();
+	$detail_page_id = $pDetailView->getPageId();
+	if ($object_id == $detail_page_id) {
+		$list_meta_keys = ["onoffice_titel" => '%1$s', "onoffice_title" => '%1$s',
```

Comment:
The way this is currently, the `%1$s`, `%2$s`, ... are magic numbers that are difficult to understand.

Since you format them all as strings anyway `$s`, what do you think of using this array to map the custom field key to the enterprise field key? So for example `"onoffice_titel" => 'objekttitel'.

Then you don't need to look in another part of the code what the different numbers mean. ;)

Reply:
Yes, I will research other solutions to solve it.